### DISCUSSION TITLE
Authenticating a webview from Supabase Flutter (or any mobile app for that matter)

### DISCUSSION THREAD

**Discussion by rphly** [supabase/supabase#28814] [Questions] [answered]
Was wondering if anyone has a good approach to automatically authenticating a user on a webview, if they are logged in on a Supabase app. Parts of my app will be delivered through a webview experience, so ideally the session would feel continuous whenever a user switches between native and web views.

Cheers

**Comment by rphly** [comment, marked as answer]
To anyone who wants to know how I solved it I just injected the supabase session object into localstorage

**Reply by JayYoon0412** [reply]
I'm having the same trouble. Can you share how you injected the session object?

**Reply by ManuLpz4** [reply]
👀
